content/index.md

```diff
@@ -97,7 +97,7 @@ Our specialties include:
   attrs={'data-block': 'rotate'}
 ) %}
 
-## Hi, I'm Miriam
+<h2>Hi, I'm Miriam</h2>
```

Comment:
Why not stay consistent with markdown here?

Reply:
I switched this because markdown was generating an anchor link on this heading